app/uk/gov/hmrc/individualsemploymentsapi/domain/integrationframework/IfId.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.domain.integrationframework
+
+import play.api.libs.functional.syntax.{unlift, _}
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+
+import scala.util.matching.Regex
+
+case class IfId(nino: Option[String], trn: Option[String])
+
+object IfId {
```

Comment:
I think this is specific to the static stub. I do not think we need this object.